lib/internal/loader/ModuleRequest.js

```diff
@@ -39,6 +38,7 @@ loaders.set('esm', async (url) => {
 const isWindows = process.platform === 'win32';
 const winSepRegEx = /\//g;
 loaders.set('cjs', async (url) => {
+  const CJSModule = require('module');
```

Comment:
Was this because of the circular reference with module again?

Reply:
yep